nolearn/lasagne/base.py

```diff
@@ -83,6 +83,17 @@ def __getstate__(self):
         return state
 
 
+def objective(layers,
+              loss_function,
+              target,
+              aggregate=aggregate,
+              **kwargs):
+    output_layer = list(layers.values())[-1]
```

Comment:
Good point.  I've made a custom subclass of `OrderedDict` for `layers_` that's more convenient to use in ab91505.